Makefile

```diff
@@ -28,7 +28,7 @@ README.html: README.md
 install: ## Install bees + libs
 install: lib src test
 	install -Dm644 lib/libcrucible.so $(PREFIX)/usr/lib/libcrucible.so
-	install -Dm755 bin/bees $(PREFIX)/usr/bin/bees
+	install -Dm755 bin/bees $(PREFIX)/usr/libexec/bees
```

Comment:
AFAIK most of the distro (Debian, Ubuntu, Arch & etc) doesn't have such path, so IMHO it's pointless to move bees

Reply:
Hmm, I'm on Gentoo... And while years back, I think I also had it in SuSE (before it became OpenSuSE). But no problem, we can stick it to sbin instead (which should still be a better place than bin). On the positive side, tho, I see that auto-completion will complete to "beesd" without be+tab=bees+d manually.

According to https://unix.stackexchange.com/questions/312146/what-is-the-purpose-of-usr-libexec other distributions would put it into /usr/lib/bees/bees instead.